packages/swayze/lib/src/widgets/default_actions/drag_n_drop_actions.dart

```diff
@@ -0,0 +1,151 @@
+import 'package:flutter/widgets.dart';
+
+import '../../../controller.dart';
+import '../../core/intents/drag_n_drop_intents.dart';
+import '../../core/viewport_context/viewport_context.dart';
+import '../../helpers/wrapped.dart';
+import '../internal_scope.dart';
+import 'default_swayze_action.dart';
+
+/// Default [Action] for [HeaderDragStartIntent].
```

Comment:
I think it would be beneficial if the docs on these actions explain a bit about their responsibility. DragStart and DragUpdate are pretty straightforward, but a bit more emphasis on the documentation on the DragEnd